letters-to-symbols/letters_to_symbols.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""letters_to_symbols.ipynb
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/1oVH4cKQI-Q_bYc6WbBNvd9OGcX93FjAu
+
+#Exercise 5
+"""
+
+def letters_to_symbols(text):
```

Comment:
This solution won't work if the string ends in a non-repeated character (e.g. "AAAABBBCCDA")